backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/model/PreferredLanguageModel.java

```diff
@@ -0,0 +1,20 @@
+package ca.gov.dtsstn.cdcp.api.web.v1.model;
+
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+import ca.gov.dtsstn.cdcp.api.web.validation.LanguageCode;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+
+@Immutable
+@JsonDeserialize(as = ImmutablePreferredLanguageModel.class)
+public interface PreferredLanguageModel {
```

Comment:
As mentioned in another comment.. this should probably be `SubscriptionUpdateModel`

Reply:
fixed